### ISSUE TITLE
请帮忙看一下，设置了5轮，跑到第二轮就自己停下来了，请问是什么原因？

### ISSUE THREAD

**Issue by deutschoo** [Bureaux-Tao/chip2020_relation_extraction#6]
你好，大佬，模型训练过程中出现了下面的错误，请你帮我看一下。如附件中的截图所示，只有loss值，没有precision,recall和f1值。设置了5轮，跑到第二轮就自己停下来了，请问是什么原因？
![图片1](https://github.com/Bureaux-Tao/chip2020_relation_extraction/assets/87679669/bf4f6bb2-15e1-4475-8a84-eaa4e2994e03)

另外，我抽取了一小部分数,20条数据作为训练集，40条数据作为测试集，进行训练。打算训练5轮，batch size为16，打算试试看代码能不能跑通。不知道这样的操作是不是导致模型跑不通了？
我的版本信息如下：
Python 3.6.13
Package              Version
-------------------- ------------
absl-py              1.4.0
astor                0.8.1
backcall             0.2.0
bert4keras           0.11.5
cached-property      1.5.2
certifi              2021.5.30
cycler               0.11.0
dataclasses          0.8
decorator            5.1.1
entrypoints          0.4
gast                 0.5.4
google-pasta         0.2.0
grpcio               1.48.2
h5py                 3.1.0
importlib-metadata   4.8.3
ipykernel            5.5.6
ipython              7.16.3
ipython-genutils     0.2.0
jedi                 0.17.2
jupyter-client       7.1.2
jupyter-core         4.9.2
Keras                2.2.4
Keras-Applications   1.0.8
Keras-Preprocessing  1.1.2
kiwisolver           1.3.1
Markdown             3.3.7
matplotlib           3.3.4
nest-asyncio         1.5.8
numpy                1.19.5
pandas               1.1.5
parso                0.7.1
pexpect              4.9.0
pickleshare          0.7.5
Pillow               8.4.0
pip                  21.2.2
prompt-toolkit       3.0.36
protobuf             3.19.6
ptyprocess           0.7.0
pydot                1.4.1
Pygments             2.14.0
pyparsing            3.1.1
python-dateutil      2.8.2
pytz                 2023.3.post1
PyYAML               6.0.1
pyzmq                25.1.2
scipy                1.5.4
setuptools           58.0.4
six                  1.16.0
tensorboard          1.14.0
tensorflow           1.14.0
tensorflow-estimator 1.14.0
termcolor            1.1.0
tornado              6.1
tqdm                 4.61.2
traitlets            4.3.3
typing_extensions    4.1.1
wcwidth              0.2.12
Werkzeug             2.0.3
wheel                0.37.1
wrapt                1.16.0
zipp                 3.6.0

**Comment by Bureaux-Tao** [comment]
加上一些trick后前几轮F1是0很正常，一般10轮以内会有数值，建议可以将早停去掉，保存最佳权重，然后手动结束训练。

**Comment by deutschoo** [comment]
> 加上一些trick后前几轮F1是0很正常，一般10轮以内会有数值，建议可以将早停去掉，保存最佳权重，然后手动结束训练。

谢谢您的回复，请问一下，运行到第二轮出现下面的情况是什么原因？如何解决
Epoch 2/20
3/3 [==============================] - 59s 20s/step - loss: 2255.7770 - global_pointer_1_loss: 26.6359 - global_pointer_2_loss: 1105.6377 - global_pointer_3_loss: 1123.5034
0it [00:00, ?it/s]
